tools/export/nb/__init__.py

```diff
@@ -325,9 +325,16 @@ def get_netbeans_file_list(file_list):
             if cur_dir and prev_dir != cur_dir:
                 # evaluate all matched items (from current and previous list)
                 matched = []
-                for element in dir_list:
-                    if element in prev_dir_list:
-                        matched.append(element)
+                # Compare the Element in Previous Dir with the Elements in Current Dir
+                # and add the Differences to the match-List
+                if len(dir_list) <= len(prev_dir_list):
```

Comment:
I prefer this implemented as:
```python
for prev_dir, cur_dir in zip(prev_dir_list, dir_list):
    if cur_dir == prev_dir:
        ...
```

With that, you don't need the if statement. `zip` will terminate when either list ends; [docs](https://docs.python.org/3/library/functions.html#zip).

Reply:
Sorry for that, didn't know that this can be done much easier! Thank you!

Reply:
No need to apologize; that's what review is for.